core/runtime/binaryen/runtime_manager.cpp

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "runtime_manager.hpp"
+
+#include <binaryen/wasm-binary.h>
+
+#include <crypto/hasher/hasher_impl.hpp>
+
+#include "runtime_external_interface.hpp"
+
+OUTCOME_CPP_DEFINE_CATEGORY(kagome::runtime::binaryen,
+                            RuntimeManager::Error,
+                            e) {
+  using kagome::runtime::binaryen::RuntimeManager;
+  switch (e) {
+    case RuntimeManager::Error::EMPTY_STATE_CODE:
+      return "Provided state code is empty, calling a function is impossible";
+    case RuntimeManager::Error::INVALID_STATE_CODE:
+      return "Invalid state code, calling a function is impossible";
+  }
+}
```

Comment:
Please process  invalid enum value.
```suggestion
  return "Unknown error";
}
```